Assets/i5 Toolkit for Unity/Runtime/OpenID Connect/Scripts/OIDC Providers/Google/GoogleOIDCProvider.cs

```diff
@@ -60,9 +60,7 @@ public void GenerateCSRFToken()
         /// <returns>Returns the access token if it could be retrieved; otherwise it returns an empty string</returns>
         public override async Task<string> GetAccessTokenFromCodeAsync(string code, string redirectUri)
         {
-            redirectUri += "code?";
-
-            EndpointsData endpoints = await InitializeEndpointsAsync();
+            await InitializeEndpointsAsync();
```

Comment:
Had to remove the "code?" since it was giving me strange errors. Also, endpoints variable was not being used so I just removed it.